src/lib/Metadata/DomMetadata.cpp

```diff
@@ -881,9 +885,9 @@ DomInfo<T>::construct() const
             entries.emplace_back("symbol", sym);
         }
 
-        if (I_.UsingName)
+        if (I_.Qualifier)
```

Comment:
This needs to `entries.emplace_back("qualifier", domCreate(I_.Qualifier, domCorpus_));`